### PR TITLE
Adding Marrison-1967-917.tsv

### PR DESCRIPTION
**Pull request by MacyL**
Hand type Marrison data. Later I will remove Marrison-1967-909.tsv

### CODE REVIEW

File: concepticondata/conceptlists/Marrison-1967-917.tsv [hunk 1]
```diff
@@ -0,0 +1,912 @@
...
+Marrison-1967-917-518	518	not	1240	NOT
+Marrison-1967-917-519	519	now	1376	NOW
+Marrison-1967-917-520	520	oath	1712	OATH
+Marrison-1967-917-521	521	to obey	1635	OBEY
+Marrison-1967-917-522	522	oil	1484	OIL (ORGANIC SUBSTANCE)
+Marrison-1967-917-523	523	old(of things)	2113	OLD (USED)
+Marrison-1967-917-524	524	old(of persons)	2112	OLD (AGED)
+Marrison-1967-917-525	525	old man	406	OLD MAN
+Marrison-1967-917-526	526	old woman	907	OLD WOMAN
+Marrison-1967-917-527	527	on	2379	UP OR ABOVE
```
[the remark this answers is not in the document]
Comment (MacyL) [reply concepticondata/conceptlists/Marrison-1967-917.tsv]: unlink the on? or all of those?

[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Marrison-1967-917.tsv]: unlink "on", as it is not well linked to "up or above", I'd say...

File: concepticondata/conceptlists/Marrison-1967-917.tsv [hunk 2]
```diff
@@ -0,0 +1,912 @@
...
+Marrison-1967-917-833	833	village	930	VILLAGE
+Marrison-1967-917-834	834	voice	1363	VOICE
+Marrison-1967-917-835	835	to vomit	1278	VOMIT
+Marrison-1967-917-836	836	waist	1681	WAIST
+Marrison-1967-917-837	837	to wait	1590	WAIT (FOR)
+Marrison-1967-917-838	838	to walk	1443	WALK
+Marrison-1967-917-839	839	wall	933	WALL (OF HOUSE)
+Marrison-1967-917-840	840	to want	1784	WANT
+Marrison-1967-917-841	841	war	935	WAR
+Marrison-1967-917-842	842	to warn	2032	WARM UP
```
[the remark this answers is not in the document]
Comment (MacyL) [reply concepticondata/conceptlists/Marrison-1967-917.tsv]: my bad.... sorry, I was exhausted so I linked to the wrong concept.

File: concepticondata/conceptlists/Link to Allen-2007-500.tsv [hunk 3]
```diff
@@ -0,0 +1 @@
+/home/wu/Documents/test_pythonvenv/bin/concepticon-data/concepticondata/conceptlists/Allen-2007-500.tsv
```
[the remark this answers is not in the document]
Comment (LinguList) [reply concepticondata/conceptlists/Link to Allen-2007-500.tsv]: ```git rm concepticondata/conceptlists/Allen-2007-500.tsv```

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-82	82	wife's father		
+Kusunda-1972-337-83	83	flower	239	FLOWER
+Kusunda-1972-337-84	84	inside	1606	INSIDE
+Kusunda-1972-337-85	85	language	1307	LANGUAGE
+Kusunda-1972-337-86	86	meat	634	MEAT
+Kusunda-1972-337-87	87	breathe (1st)	1407	BREATHE
+Kusunda-1972-337-88	88	male sexorgan		
+Kusunda-1972-337-89	89	he/ person	1211	HE
+Kusunda-1972-337-90	90	wood 	2141	TREE OR WOOD
+Kusunda-1972-337-91	91	female sex organ
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: 1836 VAGINA

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-83	83	flower	239	FLOWER
+Kusunda-1972-337-84	84	inside	1606	INSIDE
+Kusunda-1972-337-85	85	language	1307	LANGUAGE
+Kusunda-1972-337-86	86	meat	634	MEAT
+Kusunda-1972-337-87	87	breathe (1st)	1407	BREATHE
+Kusunda-1972-337-88	88	male sexorgan		
+Kusunda-1972-337-89	89	he/ person	1211	HE
+Kusunda-1972-337-90	90	wood 	2141	TREE OR WOOD
+Kusunda-1972-337-91	91	female sex organ		
+Kusunda-1972-337-92	92	wife's mother
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: new concept: mother-in-law (of male), narrower than:

http://concepticon.clld.org/parameters/1050 MOTHER-IN-LAW

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-98	98	they	817	THEY
+Kusunda-1972-337-99	99	seed	714	SEED
+Kusunda-1972-337-100	100	sweet/ sap	717	SWEET
+Kusunda-1972-337-101	101	skin	763	SKIN
+Kusunda-1972-337-102	102	feather	1201	FEATHER
+Kusunda-1972-337-103	103	egg	744	EGG
+Kusunda-1972-337-104	104	unpleasant	26	UNPLEASANT
+Kusunda-1972-337-105	105	this morning	1339	MORNING
+Kusunda-1972-337-106	106	tomorrow	1329	TOMORROW
+Kusunda-1972-337-107	107	mother's older brother
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: there's a kinship term for this, right? please check for "uncle", and add, if you don't find "uncle (older brother of mother)" or similar, we should have similar examples in concepticon already

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-99	99	seed	714	SEED
+Kusunda-1972-337-100	100	sweet/ sap	717	SWEET
+Kusunda-1972-337-101	101	skin	763	SKIN
+Kusunda-1972-337-102	102	feather	1201	FEATHER
+Kusunda-1972-337-103	103	egg	744	EGG
+Kusunda-1972-337-104	104	unpleasant	26	UNPLEASANT
+Kusunda-1972-337-105	105	this morning	1339	MORNING
+Kusunda-1972-337-106	106	tomorrow	1329	TOMORROW
+Kusunda-1972-337-107	107	mother's older brother		
+Kusunda-1972-337-108	108	tobacco used in puja	974	TOBACCO
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: what is puja? Maybe unlink, but I leave this to you to decide.

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 8]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-130	130	tall	711	TALL
+Kusunda-1972-337-131	131	now	1376	NOW
+Kusunda-1972-337-132	132	today	1283	TODAY
+Kusunda-1972-337-133	133	fever	1534	FEVER
+Kusunda-1972-337-134	134	shim		
+Kusunda-1972-337-135	135	sun	1343	SUN
+Kusunda-1972-337-136	136	night	1233	NIGHT
+Kusunda-1972-337-137	137	near here	1942	NEAR
+Kusunda-1972-337-138	138	horn	1393	HORN (ANATOMY)
+Kusunda-1972-337-139	139	a kind of  vine	343	VINE
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: you already linked VINE, this is a kind of, not hte generic term, so unlink

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-140	140	near	1942	NEAR
+Kusunda-1972-337-141	141	there	1937	THERE
+Kusunda-1972-337-142	142	root/tuber	670	ROOT
+Kusunda-1972-337-143	143	presently		
+Kusunda-1972-337-144	144	fire	221	FIRE
+Kusunda-1972-337-145	145	cry (1st)	1839	CRY
+Kusunda-1972-337-146	146	sashes
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: maybe error when typing off?

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 10]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-141	141	there	1937	THERE
+Kusunda-1972-337-142	142	root/tuber	670	ROOT
+Kusunda-1972-337-143	143	presently		
+Kusunda-1972-337-144	144	fire	221	FIRE
+Kusunda-1972-337-145	145	cry (1st)	1839	CRY
+Kusunda-1972-337-146	146	sashes		
+Kusunda-1972-337-147	147	ill	1847	SICK
+Kusunda-1972-337-148	148	new	1231	NEW
+Kusunda-1972-337-149	149	year	1226	YEAR
+Kusunda-1972-337-150	150	mustard
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: please add MUSTARD to concepticon: it occurd in Marrison as well, so you could link it in Kusunda and Marrison!

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 11]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-166	166	kodalo		
+Kusunda-1972-337-167	167	worm	1219	WORM
+Kusunda-1972-337-168	168	village	930	VILLAGE
+Kusunda-1972-337-169	169	village	930	VILLAGE
+Kusunda-1972-337-170	170	dance (1st)	1879	DANCE
+Kusunda-1972-337-171	171	learn (1st)	504	LEARN
+Kusunda-1972-337-172	172	lightening	1654	LIGHTNING
+Kusunda-1972-337-173	173	fight (1st)	1423	FIGHT
+Kusunda-1972-337-174	174	mother	1216	MOTHER
+Kusunda-1972-337-175	175	mother’s eldest sister 	1758	OLDER SISTER
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: maybe write ```mother’s``` into ```mother's```, as you don't use the normal symbol for this.

Comment (LinguList) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: In fact, generally replace ```’``` by ```'```

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 12]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-176	176	mother’s younger sister	1761	YOUNGER SISTER
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: this is not the younger sister, its some kind of aunt!

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 13]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-177	177	boy	1366	BOY
+Kusunda-1972-337-178	178	hear (1st)	1408	HEAR
+Kusunda-1972-337-179	179	water buffalo	2516	WATER BUFFALO
+Kusunda-1972-337-180	180	father’s brother’s wife		
+Kusunda-1972-337-181	181	joke		
+Kusunda-1972-337-182	182	older brother	1759	OLDER BROTHER
+Kusunda-1972-337-183	183	want (1st)	1784	WANT
+Kusunda-1972-337-184	184	deep	1593	DEEP
+Kusunda-1972-337-185	185	older sister's husband		
+Kusunda-1972-337-186	186	second son	1620	SON
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: this is not the son, better unlink!

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 14]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-204	204	cow	1007	COW
+Kusunda-1972-337-205	205	woman	962	WOMAN
+Kusunda-1972-337-206	206	you (plural)	1213	YOU
+Kusunda-1972-337-207	207	eat (1st)	1336	EAT
+Kusunda-1972-337-208	208	you (plural)	1213	YOU
+Kusunda-1972-337-209	209	Father's sister	1272	AUNT
+Kusunda-1972-337-210	210	collect (1st)	7	GATHER
+Kusunda-1972-337-211	211	ancestor	1669	ANCESTORS
+Kusunda-1972-337-212	212	grandmother	1496	GRANDMOTHER
+Kusunda-1972-337-213	213	mother's father	1383	GRANDFATHER
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: nope, this is the maternal grandfather!

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 15]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-207	207	eat (1st)	1336	EAT
+Kusunda-1972-337-208	208	you (plural)	1213	YOU
+Kusunda-1972-337-209	209	Father's sister	1272	AUNT
+Kusunda-1972-337-210	210	collect (1st)	7	GATHER
+Kusunda-1972-337-211	211	ancestor	1669	ANCESTORS
+Kusunda-1972-337-212	212	grandmother	1496	GRANDMOTHER
+Kusunda-1972-337-213	213	mother's father	1383	GRANDFATHER 
+Kusunda-1972-337-214	214	great grandfather		
+Kusunda-1972-337-215	215	grandfather	1383	GRANDFATHER
+Kusunda-1972-337-216	216	mother's mother	1496	GRANDMOTHER
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: maternal grandmother

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 16]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-217	217	man	1554	MAN
+Kusunda-1972-337-218	218	shoulder strap for net bag	2973	STRAP
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: unlink

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 17]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-273	273	arm	1673	ARM
+Kusunda-1972-337-274	274	nose	1221	NOSE
+Kusunda-1972-337-275	275	eye	1248	EYE
+Kusunda-1972-337-276	276	head	1256	HEAD
+Kusunda-1972-337-277	277	hair	1040	HAIR
+Kusunda-1972-337-278	278	leg	1297	LEG
+Kusunda-1972-337-279	279	toe	1389	TOE
+Kusunda-1972-337-280	280	finger	1303	FINGER
+Kusunda-1972-337-281	281	ear	1247	EAR
+Kusunda-1972-337-282	282	finger nail	1896	FINGERNAIL OR TOENAIL
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: fingernail, not fingernail OR toenail

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 18]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-283	283	children	2099	CHILD
+Kusunda-1972-337-284	284	stab (1st)	1434	STAB
+Kusunda-1972-337-285	285	water	948	WATER
+Kusunda-1972-337-286	286	swim (1st)	1439	SWIM
+Kusunda-1972-337-287	287	rain	658	RAIN (PRECIPITATION)
+Kusunda-1972-337-288	288	forest chicken		
+Kusunda-1972-337-289	289	chicken	1318	CHICKEN
+Kusunda-1972-337-290	290	chicken eggs	2779	CHICKEN EGG
+Kusunda-1972-337-291	291	small children	2099	CHILD
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: unlink

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 19]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-301	301	we	1212	WE
+Kusunda-1972-337-302	302	die (1st)	1494	DIE
+Kusunda-1972-337-303	303	keep (1st)	1477	KEEP
+Kusunda-1972-337-304	304	have (1st)	773	HAVE
+Kusunda-1972-337-305	305	keen	39	KEEN
+Kusunda-1972-337-306	306	bow	994	BOW
+Kusunda-1972-337-307	307	come (1st)	1446	COME
+Kusunda-1972-337-308	308	snake	730	SNAKE
+Kusunda-1972-337-309	309	lost		
+Kusunda-1972-337-310	310	point (of arrow head)	2992	TIP (OF OBJECT)
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: unlilnk, too specific

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 20]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-311	311	branch	1531	BRANCH
+Kusunda-1972-337-312	312	road	667	ROAD
+Kusunda-1972-337-313	313	day	1260	DAY (24 HOURS)
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: link to the other day (day not night)

File: concepticondata/conceptlists/Reinhard-1970-337.tsv [hunk 21]
```diff
@@ -0,0 +1,338 @@
...
+Kusunda-1972-337-318	318	bush	1683	BUSH
+Kusunda-1972-337-319	319	house	1252	HOUSE
+Kusunda-1972-337-320	320	potatos	593	POTATO
+Kusunda-1972-337-321	321	boat	1844	BOAT
+Kusunda-1972-337-322	322	stone	857	STONE
+Kusunda-1972-337-323	323	beautiful	1427	BEAUTIFUL
+Kusunda-1972-337-324	324	good	1035	GOOD
+Kusunda-1972-337-325	325	flow	2003	FLOW
+Kusunda-1972-337-326	326	girl	1646	GIRL
+Kusunda-1972-337-327	327	wither
```
Comment (LinguList) [inline concepticondata/conceptlists/Reinhard-1970-337.tsv]: why does wither occur two times in the data?

Comment (LinguList) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: check wether "wither" is compatible to be linked to "rot"

File: concepticondata/references/references.bib [hunk 22]
```diff
@@ -2112,3 +2112,10 @@ @misc{Kaiping2018
   url = {http://www.model-ling.eu/lexirumah/},
   doi = {10.5281/zenodo.1164782}
 }
+
+@book{Reinhard1970,
```
Comment (LinguList) [inline concepticondata/references/references.bib]: provide the ```address``` as well, please

File: concepticondata/conceptlists/Reinhard-1970-337.tsv (hunk 7 above)
Comment (MacyL) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: Puja is a kind of ceremony. So tobacco used in puja should be similar with incense, but the difference is that incense is made of wood powder and they used tobacco. I will unlink this one.

**Comment by MacyL** [comment]
I think I have modified both Reinhard and Marrison data. May I close this issue?

File: concepticondata/conceptlists/Reinhard-1970-337.tsv (hunk 9 above)
Comment (MacyL) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: yes, it is actually "ashes", editing now

File: concepticondata/conceptlists/Reinhard-1970-337.tsv (hunk 10 above)
Comment (MacyL) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: added

File: concepticondata/references/references.bib [hunk 23]
```diff
@@ -2119,3 +2119,4 @@ @book{Reinhard1970
 year={1970},
 address ={Kirtipur, Kathmandu, Nepal}
```
Comment (LinguList) [inline concepticondata/references/references.bib]: ```address = {Kathmandu}``` is sufficient, sorry ;)

File: concepticondata/conceptlists/Reinhard-1970-337.tsv (hunk 14 above)
Comment (MacyL) [reply concepticondata/conceptlists/Reinhard-1970-337.tsv]: linked

**Pull request merged**